torch/csrc/api/include/torch/tensor.h

```diff
@@ -7,4 +7,24 @@
 namespace torch {
 // TODO: Rename to `Tensor`.
 using Variable = autograd::Variable;
+
+/// Special "raw data" dtype.
+constexpr auto kByte = at::kByte;
```

Comment:
why isn't this kUInt8? (or whatever capitalization).  I didn't put torch.byte in pytorch because it isn't the same as Numpy (byte is signed), and I thought it could be confusing because the two dtypes (torch and np) are so similar.

I'm okay if you think that should change, but IMO we should keep pytorch and the torch namespace here the symmetric.